tasks/get-plugins.yml

```diff
@@ -7,9 +9,17 @@
     owner: root
     group: root
 
+- set_fact:
+    plugin_url: "{{ cj_nexus_url }}/{{ cj_nexus_mirror }}"
+  when: cj_nexus
```

Comment:
not the correct prefix in this role the variable prefix should be the role namespace ! which is juseppe_